tests/devices/default_qubit/test_default_qubit_native_mcm.py

```diff
@@ -378,6 +420,14 @@ def func(x):
     if shots is None and measure_f in (qml.counts, qml.sample):
         return
 
+    if measure_f == qml.probs:
+        with pytest.raises(
+            ValueError,
+            match="Cannot use qml.probs\(\) when measuring multiple mid-circuit measurements collected",
```

Comment:
This should also get rid of the pylint warning AFAIK.

```suggestion
            match=r"Cannot use qml.probs\(\) when measuring multiple mid-circuit measurements collected",
```